domain/schema/model/sql/0019-unit.sql

```diff
@@ -49,7 +49,7 @@ ON unit (name);
 CREATE INDEX idx_unit_application
 ON unit (application_uuid);
 
-CREATE INDEX idx_unit_net_node
+CREATE UNIQUE INDEX idx_unit_net_node_uuid
```

Comment:
This will prevent co-location of units on machines.

Reply:
Yes it will. In that case we'll need to make the foreign key from cloud_container table refer to unit uuid rather than net_node_uuid as we discussed, since FK constraints either need to refer to the primary key or a uniquely indexed column.